Templates/project_collection/README.md

```diff
@@ -0,0 +1,212 @@
+# Instructions for filling out Project Collection Template
+The project and data collection template contains all the information necessary
+to add a new project and/or data collection to our inventory. There are three
```

Comment:
Do we have any special instructions if someone needs to add another data collection to an existing project?

Reply:
Added instructions for this.